packages/stream_chat_flutter/lib/src/stream_chat.dart

```diff
@@ -32,21 +32,21 @@ import 'dart:ui' as ui;
 /// Use [StreamChat.of] to get the current [StreamChatState] instance.
 class StreamChat extends StatefulWidget {
   final StreamChatClient client;
-  final Widget child;
-  final StreamChatThemeData streamChatThemeData;
+  final Widget? child;
```

Comment:
child is required, can make this non nullable

Reply:
I wouldn't do it
instead I would use a default empty widget 

problem is we cant to be able to use it in the MaterialApp builder property and the passed widget is nullable
we can talk about it on a call, let me know